From 3f24430dd7517e33a02c5fac28a73cdd29bf33ae Mon Sep 17 00:00:00 2001
From: Dan Pasanen <invisiblek@cyanogenmod.org>
Date: Wed, 30 Nov 2016 18:56:46 -0600
Subject: [PATCH] [DO NOT MERGE] manually download Gello.apk to maven cache

* Until the ssl issues are fixed this pick will let people build...

Change-Id: I9ba3eb61afa65dc772c8a838768e0727f0d8cb73
---
 gello/Android.mk   |  2 ++
 gello/fix_gello.sh | 11 +++++++++++
 2 files changed, 13 insertions(+)
 create mode 100755 gello/fix_gello.sh

diff --git a/gello/Android.mk b/gello/Android.mk
index 1cf0cd0..bcdc365 100644
--- a/gello/Android.mk
+++ b/gello/Android.mk
@@ -47,5 +47,7 @@ LOCAL_MAVEN_VERSION := 40
 LOCAL_MAVEN_ARTIFACT := gello
 LOCAL_MAVEN_PACKAGING := apk
 
+$(shell vendor/cm/gello/fix_gello.sh $(LOCAL_MAVEN_VERSION))
+
 include $(BUILD_MAVEN_PREBUILT)
 endif
diff --git a/gello/fix_gello.sh b/gello/fix_gello.sh
new file mode 100755
index 0000000..0dfd613
--- /dev/null
+++ b/gello/fix_gello.sh
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+v=$1
+
+#if [ -e "~/.m2/repository/org/cyanogenmod/gello/$v/gello-$v.apk" ]; then
+  rm -rf ~/.m2/repository/org/cyanogenmod/gello
+  mkdir -p ~/.m2/repository/org/cyanogenmod/gello/$v
+  for f in gello-$v.apk gello-$v.apk.md5 gello-$v.apk.sha1; do
+    wget --no-check-certificate https://maven.cyanogenmod.org/artifactory/gello_prebuilds/org/cyanogenmod/gello/$v/$f -O ~/.m2/repository/org/cyanogenmod/gello/$v/$f
+  done
+#fi